coderd/workspaces.go

```diff
@@ -718,7 +727,7 @@ func createWorkspace(
 			Initiator(initiatorID).
 			ActiveVersion().
 			RichParameterValues(req.RichParameterValues).
-			TemplateVersionPresetID(req.TemplateVersionPresetID)
+			RunningAgentAuthTokens(agentTokensByAgentID)
```

Comment:
`agentTokensByAgentID` never seems to be populated.

Reply:
this is no longer a concern given that provisionerdserver now fetches these.